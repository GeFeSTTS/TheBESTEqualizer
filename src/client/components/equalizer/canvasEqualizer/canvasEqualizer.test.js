import React from 'react';
import { shallow } from 'enzyme';

import CanvasEqualizer from './canvasEqualizer';

describe('Grafic equaliser', () => {
  const props = {
    width: 150,
    height: 100,
    getCanvasEl: () => {},
  };

  it('should render canvas element', () => {
    const nextProps = {
      ...props,
    };
    const nextContainer = shallow(<CanvasEqualizer {
                            ...nextProps
                        }
    />);
    expect(nextContainer.find('canvas')).toHaveLength(1);
  });

  it('should render component properly', () => {
    const nextProps = {
      ...props,
    };
    const nextContainer = shallow(<CanvasEqualizer {
                            ...nextProps
                        }
    />);
    expect(nextContainer.debug()).toMatchSnapshot();
  });
});
