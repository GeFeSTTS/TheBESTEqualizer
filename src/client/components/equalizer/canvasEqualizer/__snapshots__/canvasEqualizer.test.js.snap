// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grafic equaliser should render component properly 1`] = `"<canvas className=\\"GraphicEqualiser\\" width={150} height={100} />"`;
